loom-cli/main/loom.hs

```diff
@@ -137,8 +138,33 @@ main = do
             Exit.exitWith ec
       Watch port sf ->
         watch port sf
-      Paths (PathsCommand _ _ _) ->
-        IO.hPutStrLn IO.stderr "TODO"
+      Paths (PathsCommand bundle sets _grouping) -> do
+        cwd <- getCurrentDirectory
+        config <- orDie renderLoomConfigTomlError $ resolveConfig cwd
+        loom <- resolveLoom config
+        let allResolved =
+              pure (loomResolvedConfig loom) <> (loomResolvedConfigs loom)
+            -- TODO: Component dirs
+            -- TODO: resolvePurescript?Get FilePaths from LoomFile originals
+            -- Hell; can we go *backwards* to infer a glob? or is it better to hack it in
+            pursAppPaths =
+              allResolved >>=
+                purescriptBundleFiles . loomConfigResolvedPurs
+            pursAppTestPaths =
+              allResolved >>=
+                purescriptBundleFiles . loomConfigResolvedPursTest
+            appGlobs = case bundle of
+              Purs ->
+                pursAppGlobs
+              PursTest ->
+                pursAppGlobs <> pursAppTestGlobs
+        void . flip execStateT [] $ do
```

Comment:
I know this is silly; this used to be wildly more complicated. I'll get rid of the `StateT` stuff.